src/System.Text.Utf8String/System/Text/Utf8Span.Enumerator.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+namespace System.Text.Utf8
+{
+    partial struct Utf8Span
+    {
+        public ref struct Enumerator
```

Comment:
Why do we need this type? i.e. what are the scenarios for enumerating byte by byte? If people need to access to the bytes, should they use Utf8Span.Bytes property which would return ReadOnlySpan<byte>

Reply:
Let's just remove this type.